ABaseEntity/ABaseEntity.h

```diff
@@ -52,7 +52,7 @@ class VRBASE_API ABaseEntity : public AActor, public IBaseEntity {
 
     // unrecommended to override AActor functions
     virtual void BeginPlay() override;
-    virtual void Tick(float deltaTime) override {}
+    virtual void Tick(float deltaTime) override { Super::Tick(deltaTime); }

```

Comment:
What you can do instead (since it may not be clear to anyone else viewing this PR as to why you made this change) is that once you create a PR, you can add a comment to this line stating why you made the change.

Reply:
I'll just move it to a separate branch